src/consent-manager.js

```diff
@@ -239,7 +263,7 @@ export default class ConsentManager {
             return
 
         function escapeRegexStr(str) {
-            return str.replace(/[\-\[\]\/\{\}\(\)\*\+\?\.\\\^\$\|]/g, "\\$&");
+            return str.replace(/[\-\[\]\/{}()*+?.\\^$|]/g, "\\$&");
```

Comment:
What's changing here?
Would you mind explaining the new regex?
Why did some characters get removed and others added?

Reply:
Regexp in [] are not needed to be escaped, my IDE (phpStorm) as optimized `redudant character escapings`